dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasResource.java

```diff
@@ -0,0 +1,91 @@
+package io.dockstore.webservice.resources;
+
+import java.util.Optional;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.QueryParam;
+
+import com.codahale.metrics.annotation.Timed;
+import io.dockstore.webservice.CustomWebApplicationException;
+import io.dockstore.webservice.DockstoreWebserviceConfiguration;
+import io.dockstore.webservice.core.User;
+import io.dockstore.webservice.core.WorkflowVersion;
+import io.dockstore.webservice.jdbi.WorkflowDAO;
+import io.dockstore.webservice.jdbi.WorkflowVersionDAO;
+import io.dropwizard.auth.Auth;
+import io.dropwizard.hibernate.UnitOfWork;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.Authorization;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.hibernate.SessionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static io.dockstore.webservice.Constants.JWT_SECURITY_DEFINITION_NAME;
+
+@Path("/aliases")
+@Api("/aliases")
+public class AliasResource {
+
+    private static final String OPTIONAL_AUTH_MESSAGE = "Does not require authentication for published workflows,"
```

Comment:
Can just re=use https://github.com/dockstore/dockstore/blob/33e1f07f169d0451e707586ad2fbd19b52197e4d/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java#L146 after changing visiblity (and possibly renaming)

Reply:
Where should I would put the definition?

Reply:
I'd put it in a constants file. There should already be one somewhere, maybe in the dockstore-common

Reply:
added to webservice constants file